### PR TITLE
Add redis_exporter

### PR DESCRIPTION
**Pull request by ajayk**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

### CODE REVIEW

File: redis_exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: redis_exporter
```
Comment (kaniini) [inline redis_exporter.yaml:2]: might it be helpful to call this `prometheus-redis_exporter`?

Comment (ajayk) [reply redis_exporter.yaml:2]: the upstream has redis_exporter all over , just wanted to keep the same parity , happy to change it

File: prometheus-redis_exporter.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: prometheus-redis_exporter
```
Comment (imjasonh) [inline prometheus-redis_exporter.yaml:2]: Should we name this -exporter? I don't think apk cares, but just for consistency

Comment (ajayk) [reply prometheus-redis_exporter.yaml]: Addressed

**Review by imjasonh — APPROVED** [review]

**Pull request merged**